plugin/db/mysql/restore.go

```diff
@@ -50,6 +51,11 @@ const (
 	LetterCaseOnDiskLowerCaseCmp = 2
 )
 
+var (
+	binlogFileTasks = make(map[string]bool)
+	muBinlogFile    sync.RWMutex
```

Comment:
We only use `Lock()` and `Unlock()`. Can we use `sync.Mutex`?

Reply:
SG.